awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupAdd/InventoryGroupAdd.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import { withI18n } from '@lingui/react';
+import { withRouter } from 'react-router-dom';
+import { GroupsAPI } from '@api';
+
+import ContentError from '@components/ContentError';
+import InventoryGroupForm from '../InventoryGroupForm/InventoryGroupForm';
+
+function InventoryGroupsAdd({ history, inventory, setBreadcrumb }) {
+  useEffect(() => setBreadcrumb(inventory), [inventory, setBreadcrumb]);
+  const [error, setError] = useState(null);
+  const handleSubmit = async values => {
+    values.inventory = inventory.id;
+    try {
+      const { data } = await GroupsAPI.create(values);
+      history.push(`/inventories/inventory/${inventory.id}/groups/${data.id}`);
+    } catch (err) {
+      setError(err);
+    }
+  };
+  const handleCancel = () => {
+    history.push(`/inventories/inventory/${inventory.id}/groups`);
+  };
+  if (error) {
+    return <ContentError />;
```

Comment:
Form submission errors should show be displayed on the Form. It looks like we are showing a <ContentError> and we're passing the error to the InventoryGroupForm. Are we showing a form submission error in 2 places?

Reply:
Yes, that is what is happening.  Consider if fixed.